modules/luci-mod-system/htdocs/luci-static/resources/view/system/led-trigger/heartbeat.js

```diff
@@ -1,10 +1,19 @@
 'use strict';
 'require baseclass';
+'require form';
 
 return baseclass.extend({
 	trigger: _('Heartbeat interval (kernel: heartbeat)'),
 	description: _('The LED flashes to simulate actual heart beat.') +
 		_('The frequency is in direct proportion to 1-minute average CPU load.'),
 	kernel: true,
-	addFormOptions: function(s) {}
+	addFormOptions: function(s) {
+		var o;
+
+		o = s.option(form.Flag, 'inverted', _('Invert blinking'),
+			_('The LED is more on then off'));
```

Comment:
I would suggest the following phrase:
`When enabled, the LED is lit continuously and flickering on activity. When disabled, the LED is off by default and only flashes on activity.`